mcs/class/System.Core/System.IO.Pipes/NamedPipeClientStream.cs

```diff
@@ -106,13 +110,17 @@ public NamedPipeClientStream (string serverName, string pipeName, PipeAccessRigh
 			if (impersonationLevel != TokenImpersonationLevel.None ||
 			    inheritability != HandleInheritability.None)
 				throw ThrowACLException ();
-#if MOBILE
+#if MOBILE && !UNITY
 			throw new NotImplementedException ();
 #else
 			if (IsWindows)
 				impl = new Win32NamedPipeClient (this, serverName, pipeName, desiredAccessRights, options, inheritability);
 			else
+#if UNITY
```

Comment:
Do you want UNITY_AOT here instead? This will affect unityjit as well.

Reply:
Oh yes, good point. I'll change these to be `UNITY_AOT`.